FrameworkSystem/DB/ProxyDB.py

```diff
@@ -265,7 +265,7 @@ def completeDelegation( self, requestId, userDN, delegatedPem ):
     retVal = chain.isValidProxy( ignoreDefault = True )
     noGroupFlag = False
     if not retVal[ 'OK' ]:
-      if retVal['Message'] == "Proxy does not have an explicit group":
+      if "Proxy does not have an explicit group" in retVal['Message']:
```

Comment:
Actually the error returned was not adequate ( generic error with a specific message ). In this case the only way to check is to make a direct string comparison. I have created a more specific error that can be useful in other places as well. In this case the cmpError use is straightforward.